addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -308,12 +388,48 @@ odoo.define('website_slides.quiz', function (require) {
                         self.slide.completed = true;
                         self.trigger_up('slide_completed', {slide: self.slide, completion: data.channel_completion});
                     }
+                    self._hideEditOptions();
                     self._renderAnswersHighlighting();
                     self._renderValidationInfo();
                 }
             });
         },
 
+        /**
+         * Get all the question information after clicking on
+         * the edit button
+         * @param $elem
+         * @returns {{id: *, sequence: number, text: *, answers: Array}}
+         * @private
+         */
+        _getQuestionDetails: function ($elem) {
+            var answers = [];
+            $elem.find('.o_wslides_quiz_answer').each(function () {
+                answers.push({
+                    'id': $(this).data('answerId'),
+                    'text_value': $(this).data('text'),
+                    'is_correct': $(this).data('isCorrect')
+                });
+            });
+            return {
+                'id': $elem.data('questionId'),
+                'sequence': parseInt($elem.find('.o_wslides_quiz_question_sequence').text()),
+                'text': $elem.data('title'),
+                'answers': answers,
+            };
+        },
+
+        /**
+         * If the slides has been called with the Add Quiz button on the slide list
+         * it goes straight to the 'Add Quiz' button and clicks on it.
+         * @private
+         */
+        _checkLocationHref: function() {
+            if (window.location.href.includes('?quiz_quick_create')) {
```

Comment:
You can probably remove the question mark, so it will still work even if it's passed as the second parameter.